test/Microsoft.ML.Predictor.Tests/TestPredictors.cs

```diff
@@ -1035,25 +1035,38 @@ public IList<TestDataset> GetDatasetsForCalibratorTest()
         }
 
         /// <summary>
-        ///A test for calibrators
+        ///A test for no calibrators
+        ///</summary>
+        [Fact]
+        [TestCategory("Calibrator")]
+        public void DefaultCalibratorPerceptronTest()
+        {
+            var datasets = GetDatasetsForCalibratorTest();
+            RunAllTests( new[] { TestLearners.perceptronDefault }, datasets, new string[] { "cali={}" }, "nocalibration");
+            Done();
+        }
+
+        /// <summary>
+        ///A test for PAV calibrators
         ///</summary>
         [Fact(Skip = "Need CoreTLC specific baseline update")]
         [TestCategory("Calibrator")]
-        public void CalibratorPerceptronTest()
+        public void PAVCalibratorPerceptronTest()
         {
             var datasets = GetDatasetsForCalibratorTest();
-            RunAllTests(
-                new[] { TestLearners.perceptronDefault },
-                datasets,
-                new string[] { "cali={}" }, "nocalibration");
-            RunAllTests(
-                new[] { TestLearners.perceptronDefault },
-                datasets,
-                new[] { "cali=PAV" }, "PAVcalibration");
-            RunAllTests(
-            new[] { TestLearners.perceptronDefault },
-                datasets,
-                new string[] { "numcali=200" }, "calibrateRandom");
+            RunAllTests( new[] { TestLearners.perceptronDefault }, datasets, new[] { "cali=PAV" }, "PAVcalibration");
+            Done();
+        }
+
+        /// <summary>
+        ///A test for random calibrators
+        ///</summary>
+        [Fact]
+        [TestCategory("Calibrator")]
+        public void RandomCalibratorPerceptronTest()
+        {
+            var datasets = GetDatasetsForCalibratorTest();
+            RunAllTests( new[] { TestLearners.perceptronDefault }, datasets, new string[] { "numcali=200" }, "calibrateRandom");
```

Comment:
I know you didn't do this, but we might want to take advantage of the opportunity we have now to correct the disparity here. That is, calibration is the prefix here, and the suffix elsewhere. Not a big deal, don't consider this comment blocking...